entries/d/didiglobal.com.json

```diff
@@ -0,0 +1,31 @@
+{
+  "Didi": {
+    "domain": "didiglobal.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Didi decides when 2FA is required; user cannot enable 2FA for every login.",
+    "categories": [
+      "transport"
+    ],
+    "regions": [
```

Comment:
Do you know if the [Brazilian site](https://99app.com/) and the [Japanese site](https://didimobility.co.jp/) use the same authentication backend?

Reply:
Im not sure, finding information on other countries seems to be very difficult, and their global website is very limited

Reply:
Gotcha. I'll do some more research.

Reply:
If its known as 99 in Brazil, would I need to create a separate json? and icon?

Reply:
If it uses the same authentication backend, that is, you can sign in with the same account, then no. You can add the domain as an [additional domain](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties).

If it uses a separate account system, then you should create a new entry for that.